server/src/database/models/Order.ts

```diff
@@ -0,0 +1,35 @@
+import { DataTypes } from 'sequelize';
+import sequelize from '../config/connection';
+
+const Order = sequelize.define('orders', {
+id:{
+    type: DataTypes.INTEGER,
+    autoIncrement: true,
+    primaryKey: true,
+},
+date:{
+    type: DataTypes.DATE,
+    allowNull: false
+},
+total_price:{
+    type: DataTypes.DECIMAL,
+    allowNull: false
+},
+paid_price:{
+    type: DataTypes.DECIMAL,
+    allowNull: false
+},
+status:{
+    type: DataTypes.STRING,
+    allowNull: false
+},
+supplier:{
+    type: DataTypes.STRING,
+},
+is_admin:{
+    type: DataTypes.BOOLEAN,
+    allowNull: false,
+},
```

Comment:
I think there is a problem here 
it's not like the ER diagram

```suggestion
is_supplied:{
    type: DataTypes.BOOLEAN,
    allowNull: false,
},
```